cmd/csalt/main.go

```diff
@@ -82,6 +83,12 @@ func (devQ *DeviceQuery) UnmarshalText(b []byte) error {
 type Args struct {
 	DeviceInfo DeviceQuery `arg:"positional"`
 	Commands   []string    `arg:"positional"`
+	Show       bool        `arg:"-s" help:"Print salt ids for device names"`
+	Server     string      `help:"--server to use, this should be defined in cacophony-user.yaml"`
+	TestServer bool        `arg:"--test" help:"Connect to the test api server"`
+	LiveServer bool        `arg:"--live" help:"Translate device names to salt ids"`
```

Comment:
I don't understand what this does?

Reply:
wrong help message, but if you have changed your default server in the config, you can use this to connect to api.cacophony.org.nz

Reply:
Ah right, so "live" is "prod"   

Maybe call it `--prod` since that's what we tend to call this environment elsewhere